.gitignore

```diff
@@ -37,3 +37,8 @@
 
 # rats-devtools generates these files for us
 bin/*.pipx
+
+# ignore everything except *.py and readme.md under tests/python/jupytext
+rats-processors/test/python/jupytext/**
+!rats-processors/test/python/jupytext/*py
+!rats-processors/test/python/jupytext/readme.md
```

Comment:
I would not put the scripts in tests, rather `docs`, and call them from `test`?

They are not pytest, so I would not put them there, it is confusing...

Reply:
Also, let's meet tomorrow and you show me what you created, and we should sync with Lorenzo to incorporate these conversions when building the docs?

Explain to me also how the `jupytext` package works...

Reply:
* Moved *.py files to docs/_jupytext_tutorials.
* Created unit tests that executes them.
* Added command `rats-devtools build-jupytext-notebooks` that builds *.md notebooks from the *.py files, places them under docs/notebooks, and links to them.
* Like `rats-devtools build_api_docs`, the new command is not triggered by `rats-devtools mkdocs-serve`, but is triggered in CI.  Locally, you'd need to manually run it before `rats-devtools mkdocs-serve`.